apps/client/src/layout/components/top-navigation/top-navigation.tsx

```diff
@@ -23,7 +23,7 @@ export function TopNavigation() {
   }
 
   return (
-    <>
+    <div id="h" style={{ position: 'sticky', top: 0, zIndex: 1002 }}>
```

Comment:
Why the arbitrary z-index? Why not make it z-index 1?

Reply:
cloudscape recommends minimum 1002 - https://github.com/awslabs/iot-application/pull/174/files

Reply:
What stacking context is the top nav in?

Reply:
Either I don't actually understand stacking context or https://cloudscape.design/get-started/dev-guides/z-index/ makes no sense.

Reply:
It's written from the perspective that there's a single stacking context for the entire app?

Reply:
How does this affect the dashboard-page? I think the ideal layout for that page is one where the document doesn't scroll at all. That would cause a problem with the webgl context rendering data points in the right spot.

Reply:
this makes the document not scroll at all on the dashboard page.

Reply:
👍 perfect